### ISSUE TITLE
Entity names shrinks to the shortes entity in the group

### ISSUE THREAD

**Issue by prohor33** [natasha/ipymarkup#5]
Hi! I found some small bug.
Here is the example:

Top entity is short, so it's name. 3 other entities are pretty long, but their names are shrinked to the shortest (top) entity.
![image](https://user-images.githubusercontent.com/5472900/82837147-e7595900-9ed0-11ea-8afc-02bbba068c29.png)

The same example with bigger line_gap:
![image](https://user-images.githubusercontent.com/5472900/82837400-96963000-9ed1-11ea-9f4f-ad85971da507.png)

If we make top entity longer all renders fine:
![image](https://user-images.githubusercontent.com/5472900/82837428-aca3f080-9ed1-11ea-9662-e0c8ad736f12.png)

What expected to be correct behaviour: each name shrinkage corresponding to its entity length, not to the shortest entity in the group.
Thanks!

**Comment by prohor33** [comment]
Also, could we make shrinkage with new lines, not with overlaps? Thx!

**Comment by kuk** [comment]
Thank you for bug report. Fixed in 0.9.0 `pip install ipymarkup>=0.9.0`

```
from ipymarkup import show_span_line_markup as show_markup

text = 'ИП Горбатов'
spans = [
    (0, 2, 'ОПФ'),
    (0, 10, 'Уча PER Кре'),
    (0, 10, 'Уча PER улк'),
]
show_markup(text, spans)
```
![image](https://user-images.githubusercontent.com/153776/82872352-9c286000-9f3b-11ea-9a01-acedd200a1a9.png)
